origin.spec

```diff
@@ -35,6 +35,9 @@
 %global make_redistributable 1
 %endif
 
+# by default build the test binaries for Origin
+%global build_tests 1
+
```

Comment:
Is that better practice than what we have now? I was following how we define `make_redistributable` above, and we seem to be able to [toggle it](https://github.com/openshift/origin/blob/master/Makefile#L251) via `rpmbuild` options like you say.